app/routes/index.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
```

Comment:
It looks like we want a `reminders` route for this logic to live in. The index route is for overarching stuff that applies to all routes. Generate a reminders route, adjust the path to match '/', and move this model hook into that route instead.